src/cursor/cur_layered.c

```diff
@@ -251,10 +251,12 @@ __clayered_open_stable(WT_CURSOR_LAYERED *clayered, bool leader)
     WT_SESSION_IMPL *session;
     const char *cfg[4] = {WT_CONFIG_BASE(CUR2S(clayered), WT_SESSION_open_cursor), "", NULL, NULL};
     const char *checkpoint_name, *stable_uri;
+    wt_timestamp_t last_ckpt_timestamp;
 
     session = CUR2S(clayered);
     layered = (WT_LAYERED_TABLE *)clayered->dhandle;
     stable_uri = layered->stable_uri;
+    last_ckpt_timestamp = S2C(session)->disaggregated_storage.last_checkpoint_timestamp;
```

Comment:
I think this needs to be an ordered read.

Reply:
This logic is not a part of this function anymore.